lib/usd/ui/views/ItemDelegate.cpp

```diff
@@ -43,7 +45,26 @@ QWidget* ItemDelegate::createEditor(QWidget* parent, const QStyleOptionViewItem&
 		assert(treeItem != nullptr);
 		if (nullptr != treeItem)
 		{
-			VariantsEditorWidget* editor = new VariantsEditorWidget(parent, this, treeItem->prim());
+			// Look in the import data (from the tree model) to see if it has a prim variant selection
```

Comment:
When creating the variant set selection combo boxes, we'll look at import data for matching items to set from.